src/components/molecules/event/eventBody/EventBody.tsx

```diff
@@ -205,9 +213,9 @@ export const EditEvent: React.FC<{ event: Event; setEdit: () => void }> = ({
                     }
                     label={'Transportation'}></ToggleButton>
                   <ToggleButton
-                    initValue={toggleActive}
-                    onChange={() => setToggleActive(!toggleActive)}
-                    label={toggleActive ? 'Active' : 'Inactive'}></ToggleButton>
+                    initValue={togglePublic}
+                    onChange={() => setTogglePublic(!togglePublic)}
+                    label={togglePublic ? 'Public' : 'Private'}></ToggleButton>
```

Comment:
Norwegian label?

Reply:
yes should be Norwegian, admin pages are a little less restrictive :call_me_hand: added fix now

Reply:
🥳 just merge when ready then